### PR TITLE
Set modules and dependency versions

### PR DESCRIPTION
**Pull request by sivaschenko**
Set modules and dependency versions

### CODE REVIEW

File: AdobeStockAdminUi/composer.json [hunk 1]
```diff
@@ -3,11 +3,11 @@
     "description": "Magento module responsible for the admin panel UI implementation",
     "require": {
         "php": "~7.1.3||~7.2.0||~7.3.0",
-        "magento/framework": "*",
-        "magento/module-adobe-ims-api": "*",
-        "magento/module-adobe-stock-client-api": "*",
-        "magento/module-backend": "*",
-        "magento/module-config": "*"
+        "magento/framework": "102.0.*",
+        "magento/module-adobe-ims-api": "100.0.*",
```
Comment (YevSent) [inline AdobeStockAdminUi/composer.json]: PHP Interface (marked with @api) | Inject in a constructor and/or call methods | MAJOR
https://devdocs.magento.com/guides/v2.3/extension-dev-guide/versioning/dependencies.html#determine-module-dependency

File: AdobeStockAdminUi/composer.json [hunk 2]
```diff
@@ -3,11 +3,11 @@
...
+        "magento/module-adobe-stock-client-api": "100.0.*",
```
Comment (YevSent) [inline AdobeStockAdminUi/composer.json]: PHP Interface (marked with @api) | Inject in a constructor and/or call methods | MAJOR
https://devdocs.magento.com/guides/v2.3/extension-dev-guide/versioning/dependencies.html#determine-module-dependency

File: AdobeStockAsset/composer.json [hunk 3]
```diff
@@ -3,12 +3,12 @@
...
         "php": "~7.1.3||~7.2.0||~7.3.0",
-        "magento/framework": "*",
-        "magento/module-adobe-stock-asset-api": "*",
-        "magento/module-adobe-stock-client-api": "*",
-        "magento/module-adobe-media-gallery-api": "*",
-        "magento/module-adobe-media-gallery": "*",
-        "magento/module-config": "*"
+        "magento/framework": "102.0.*",
+        "magento/module-adobe-stock-asset-api": "100.0.*",
+        "magento/module-adobe-stock-client-api": "100.0.*",
```
Comment (YevSent) [inline AdobeStockAsset/composer.json]: PHP Interface (marked with @api) | Inject in a constructor and/or call methods | MAJOR
https://devdocs.magento.com/guides/v2.3/extension-dev-guide/versioning/dependencies.html#determine-module-dependency

File: AdobeStockAsset/composer.json [hunk 4]
```diff
@@ -3,12 +3,12 @@
...
+        "magento/module-adobe-media-gallery-api": "100.0.*",
```
Comment (YevSent) [inline AdobeStockAsset/composer.json]: PHP Interface (marked with @api) | Inject in a constructor and/or call methods | MAJOR
https://devdocs.magento.com/guides/v2.3/extension-dev-guide/versioning/dependencies.html#determine-module-dependency

Also, I suppose this package should be moved to the core.

File: AdobeStockAsset/composer.json [hunk 5]
```diff
@@ -3,12 +3,12 @@
...
+        "magento/module-adobe-media-gallery": "100.0.*",
```
Comment (YevSent) [inline AdobeStockAsset/composer.json]: Why this package has dependencies on both API and non-API packages?

File: AdobeStockClient/composer.json [hunk 6]
```diff
@@ -3,10 +3,10 @@
     "description": "Magento module responsible for interaction with Adobe Stock API implementation",
     "require": {
         "php": "~7.1.3||~7.2.0||~7.3.0",
-        "magento/framework": "*",
-        "magento/module-adobe-ims-api": "*",
-        "magento/module-adobe-stock-client-api": "*",
-        "magento/module-config": "*",
+        "magento/framework": "102.0.*",
+        "magento/module-adobe-ims-api": "100.0.*",
```
Comment (YevSent) [inline AdobeStockClient/composer.json]: PHP Interface (marked with @api) | Inject in a constructor and/or call methods | MAJOR
https://devdocs.magento.com/guides/v2.3/extension-dev-guide/versioning/dependencies.html#determine-module-dependency

File: AdobeStockImage/composer.json [hunk 7]
```diff
@@ -3,14 +3,14 @@
...
     "require": {
         "php": "~7.1.3||~7.2.0||~7.3.0",
-        "magento/framework": "*",
-        "magento/module-adobe-stock-client-api": "*",
-        "magento/module-adobe-stock-asset-api": "*",
-        "magento/module-adobe-stock-image-api": "*",
-        "magento/module-adobe-media-gallery-api": "*"
+        "magento/framework": "102.0.*",
+        "magento/module-adobe-stock-client-api": "100.0.*",
+        "magento/module-adobe-stock-asset-api": "100.0.*",
```
Comment (YevSent) [inline AdobeStockImage/composer.json]: PHP Interface (marked with @api) | Inject in a constructor and/or call methods | MAJOR
https://devdocs.magento.com/guides/v2.3/extension-dev-guide/versioning/dependencies.html#determine-module-dependency

File: AdobeStockImage/composer.json [hunk 8]
```diff
@@ -3,14 +3,14 @@
...
+        "magento/module-adobe-stock-image-api": "100.0.*",
+        "magento/module-adobe-media-gallery-api": "100.0.*"
```
Comment (YevSent) [inline AdobeStockImage/composer.json]: PHP Interface (marked with @api) | Inject in a constructor and/or call methods | MAJOR
https://devdocs.magento.com/guides/v2.3/extension-dev-guide/versioning/dependencies.html#determine-module-dependency

File: AdobeStockImageAdminUi/composer.json [hunk 9]
```diff
@@ -3,17 +3,17 @@
...
-        "magento/module-adobe-stock-image-api": "*",
-        "magento/module-adobe-stock-client-api": "*",
-        "magento/module-backend": "*",
-        "magento/module-ui": "*"
+        "magento/framework": "102.0.*",
+        "magento/module-adobe-ims": "100.0.*",
+        "magento/module-adobe-ims-api": "100.0.*",
+        "magento/module-adobe-stock-asset-api": "100.0.*",
+        "magento/module-adobe-stock-image-api": "100.0.*",
+        "magento/module-adobe-stock-client-api": "100.0.*",
```
Comment (YevSent) [inline AdobeStockImageAdminUi/composer.json]: PHP Interface (marked with @api) | Inject in a constructor and/or call methods | MAJOR
https://devdocs.magento.com/guides/v2.3/extension-dev-guide/versioning/dependencies.html#determine-module-dependency

**Review by YevSent — CHANGES_REQUESTED** [review]

File: AdobeIms/composer.json [hunk 10]
```diff
@@ -22,5 +22,6 @@
         "psr-4": {
             "Magento\\AdobeIms\\": ""
         }
-    }
+    },
+    "version": "100.0.0"
```
Comment (YevSent) [inline AdobeIms/composer.json]: As those packages are not core modules, I would suggest using `1.0.0` as a version.

File: AdobeStockAsset/composer.json (hunk 4 above)
Comment (sivaschenko) [reply AdobeStockAsset/composer.json]: AdobeMediaGallery and AdobeUi modules will be removed from this repo after they will be merged to core

File: AdobeStockAsset/composer.json (hunk 5 above)
Comment (sivaschenko) [reply AdobeStockAsset/composer.json]: Non-API package is installing a database table that is referenced in AdobeStockAsset db table foreign key

File: AdobeIms/composer.json (hunk 10 above)
Comment (sivaschenko) [reply AdobeIms/composer.json]: Changed

**Comment by sivaschenko** [comment]
@joni-jones versions are updated according to review comments. Please review

**Review by YevSent — APPROVED** [review]

**Pull request merged**